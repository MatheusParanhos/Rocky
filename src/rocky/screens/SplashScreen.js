import React, { Component } from 'react'
import { Text, View, StyleSheet, TimerMixin } from 'react-native'
import { withNavigation } from "react-navigation";

export class Splash extends Component {
    componentDidMount() {
        this.navigateToApp()
    }
    navigateToApp(){
        this.timer = setTimeout(() => {
            this.props.navigation.navigate("App");
          }, 2000);
    }
  render() {
    return (
      <View style={styles.container}>
        <Text> Splash </Text>
      </View>
    )
  }
}

const styles = StyleSheet.create({
    container: {
      flex: 1,
      justifyContent: "center",
      alignItems: "center",
      backgroundColor: 'white'
    }
  });

  export default withNavigation(Splash)