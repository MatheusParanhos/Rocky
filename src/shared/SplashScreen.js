import React, { Component } from "react";
import { Text, View, StyleSheet } from "react-native";
import { colors } from "../theme/theme";
export class Splash extends Component {
  componentDidMount() {
    console.log(this.props);
    this.navigateToApp();
  }
  navigateToApp() {
    this.timer = setTimeout(() => {
      this.props.navigation.navigate("App");
    }, 2000);
  }
  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.text}> Rocky! </Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: colors.primary
  },
  text: {
    fontSize: 100,
    fontFamily: "SignatraDEMO",
    color: colors.white
  }
});

export default Splash;
