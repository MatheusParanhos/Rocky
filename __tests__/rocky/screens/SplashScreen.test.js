import React from 'react';
import SplashScreen from '../../../src/rocky/screens/SplashScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<SplashScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});