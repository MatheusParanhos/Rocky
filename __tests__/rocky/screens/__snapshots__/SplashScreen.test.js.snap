// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
     textInComponent 
  </Text>
</View>
`;
